File: Information 7-9/JS/bus.js
function enter(){
	var Child   = Number($('.children').val()),
			parents = Number($('.parents').val()),
			bus			= Number($('.k_index').val());
	if(Child < 0 || Child == 0 || Child == '' && parents < 2 || parents == 0 || parents == ''){
		alert('Error');
	}
	else{
	if((Child + parents) % bus != 0){
		console.log(Child + "+" + parents + '=' + Child + parents);
		alert('0');
	}
	else{
		alert((Child + parents) / bus);
	}
	}
}

File: Information 7-9/informatics/bus.cpp
#include "stdafx.h"
#include <iostream>
#include <conio.h>
#include <cmath>

using std::cout;
using std::cin;
using std::endl;


int main()
{
	setlocale(LC_ALL, "Russian");
	int child;
	int parents;
	int bus;
	cout << "Дети:";cin >> child;
	cout << "Взрослые:";cin >> parents;
	cout << "Вместимость автобуса:";cin >> bus;
	if (child <= 0 && parents < 2 || parents == 0) {
		cout << "Error";
	}
	else {
		if ((child + parents) % bus != 0) {
			cout << 0;
		}
		else {
			cout << ((child + parents) / bus);
		}
	}


	_getch();
	return 0;
}

